docs/python/building.py

```diff
@@ -0,0 +1,51 @@
+class Building(object):
+	"""Class representing all the data for a building
+	
+	'attribute name': 'type'
+	swagger_types = {
+		'status': 'str',
+		'buildingId': 'str',
+		'nameList': 'list[str]',
+		'numWashers': 'int',
+		'numDryers': 'int',
+	}
+	
+	'attribute name': 'Attribute name in Swagger Docs'
+	attribute_map = {
+		'status': 'status',
+		'buildingId': 'buildingId',
+		'nameList': 'nameList',
+		'numWashers': 'numWashers',
+		'numDryers': 'numDryers'
+	}
+	"""
+	
+	def __init__(self, status, buildingId, nameList, numWashers, numDryers):
+		"""Class instatiation
+		
+		Check if all the attributes are valid and assigns them if they are
+		Raises ValueError if attributes are invalid
+		"""
+		allowed_status = ["success", "failure"]
```

Comment:
Be consistent in your variable names, in this case you'd want `allowedStatuses` or similar. I made it plural because it's a list.